infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -468,7 +477,6 @@ private synchronized void startSnapshotApply(LogReplication.LogReplicationEntryM
         setDataConsistentWithRetry(false);
         snapshotWriter.startSnapshotSyncApply();
         completeSnapshotApply(entry);
-        setDataConsistentWithRetry(true);
```

Comment:
looks like this method is no longer used anywhere and can now be removed altogether?

Reply:
oh no.. it is used to set the isConsistent to false.  Please ignore the above comment.